grafana-plugin/src/components/Table/Table.tsx

```diff
@@ -33,28 +33,35 @@ export interface Props<RecordType = unknown> extends TableProps<RecordType> {
 }
 
 const GTable: FC<Props> = (props) => {
-  const { columns, data, className, pagination, loading, rowKey, expandable, ...restProps } = props;
+  const { columns, data, className, pagination, loading, rowKey, expandable: expandableProp, ...restProps } = props;
 
   const { page, total: numberOfPages, onChange: onNavigate } = pagination || {};
 
-  if (expandable) {
-    expandable.expandIcon = ({ expanded, record }) => {
-      return (
-        <div className={cx('expand-icon', { [`expand-icon__expanded`]: expanded })}>
-          <ExpandIcon />
-        </div>
-      );
-    };
-  }
+  const expandable = useMemo(() => {
+    return expandableProp
```

Comment:
Just a suggestion to help with naming as I see you've renamed `expandable` to `expandableProp`. There's a naming convention you can append `fn` to functions to let the reader know it's a function, therefore you can have `expandableFn` which is the memo, and `expandable` as a prop.

Reply:
fixed